docs/integration_tests.rst

```diff
@@ -1,31 +1,38 @@
-# Integration Test Setup
+Integration Test Setup
```

Comment:
Is there a reason why you switch to rst, is this in preparation for sphinx?

Reply:
@tkilias yes, I thought this is a first step into the direction of using sphinx. Will simplify the transition, also the README is/was already in the `.rst` format.

Reply:
Ok